boa/src/exec/switch/mod.rs

```diff
@@ -0,0 +1,34 @@
+use super::{Executable, Interpreter};
+use crate::{
+    builtins::value::{ResultValue, Value},
+    syntax::ast::node::Switch,
+};
+
+impl Executable for Switch {
```

Comment:
If I understand this correctly, this implements the `switch` statement execution, right? This was not previously present, right?

I guess that the only thing left is to `break` when needed, right? Could we add a `// TODO` here?

Reply:
This was already present (it was within the node file), I've added a TODO and will create an issue so that it can be fixed.

Reply:
Created issue #439